src/java/test/org/apache/zookeeper/server/persistence/FileTxnSnapLogTest.java

```diff
@@ -113,6 +116,26 @@ public void testAutoCreateDb() throws IOException {
         attemptAutoCreateDb(dataDir, snapDir, sessions, priorAutocreateDbValue, "false", 0L);
     }
 
+    @Test
+    public void testGetTxnLogSyncElapsedTime() throws IOException {
```

Comment:
Done. It looks like this one, so any suggestion to remove duplicate or improve it is welcome.